### DISCUSSION TITLE
Create Trunk, Error: getaddrinfo ENOTFOUND sipproxy.fonoster

### DISCUSSION THREAD

**Discussion by gcareri** [fonoster/yaps#565] [Q&A]
When I try to create a trunk, I get the following message: 

_2 UNKNOWN: Error: getaddrinfo ENOTFOUND sipproxy.fonoster_

This is the **HTTP request**:

POST https://console.fonoster.io/api/sdk/providers {}

This is the **HTTP response**:
{
    "status": 500,
    "title": "Internal Server Error",
    "message": "2 UNKNOWN: Error: getaddrinfo ENOTFOUND sipproxy.fonoster",
    "success": false,
    "timestamp": 1700113925994
}

Could you help me to solve? Thanks in advance.

**Comment by psanders** [comment]
Are you still experiencing this issue, @gcareri?

**Reply by gcareri** [reply]
Hi @psanders ,

it works, but I don't receive any sip message in my side. 

Could you send me the list of ip addrs that I have to open in my firewall, please?

Thanks in advance.

**Reply by psanders** [reply]
Sorry for the late response; 

You should be getting an error on the console.

If you need to add an IP to your firewall that would be `157.245.118.236`
